src/smartHandler.ts

```diff
@@ -167,8 +167,12 @@ export class SMARTHandler implements Authorization {
 
         if (fhirUserClaim && usableScopes.some((scope) => scope.startsWith('user/'))) {
             userIdentity.fhirUserObject = getFhirUser(fhirUserClaim);
-            if (patientOrgsClaim) {
-                userIdentity.patientOrgs = getFhirResource(patientOrgsClaim, fhirServiceBaseUrl);
+            if (patientOrgsClaim && patientOrgsClaim.length) {
```

Comment:
We should probably throw an error if the parsed value is not an array. We have similar checks to confirm any FHIR resource reference is in the correct format. For example, we could throw UnAuthorizedError like so:
```
throw new UnauthorizedError('patientOrgs claim contains incorrect data type value');
```